cdap-ui/app/directives/dag-plus/my-dag-ctrl.js

```diff
@@ -605,8 +605,19 @@ angular.module(PKG.name + '.commons')
 
     vm.onNodeDelete = function (event, node) {
       event.stopPropagation();
-      DAGPlusPlusNodesActionsFactory.removeNode(node.name);
-      vm.instance.remove(node.name);
+      let modal = $uibModal.open({
+        templateUrl: 'dag-plus/confirm-deletion-modal.html',
+        size: 'sm',
+        backdrop: true,
+        keyboard: true,
+        windowTopClass: 'delete-modal',
```

Comment:
this class should be `confirm-modal hydrator-modal center`. Then you can wrap the buttons on the footer with another div with `pull-right`.

see `/templates/create/popovers/run-preview-action-confirmation-modal.html`